src/components/expense-list-section/expense-list/expense-list.tsx

```diff
@@ -0,0 +1,66 @@
+import { ExpenseItem } from "./expense-item";
+import { ListProps } from "../../../@types/expense-list-prop";
+import { ExpensesHeader } from "./expenses-header";
+import { ExpenseListHeader } from "./expense-list-header";
+import "./expense-list.css";
+
+export const ExpenseList = ({ list }: { list: ListProps }) => {
+  {
+    /* TODO: */
+  }
+  // const [sortOrder, setSortOrder] = useState<"asc" | "desc">("desc");
+  // const [currentPage, setCurrentPage] = useState(1);
+  // const {
+  //   data: expenses,
+  //   isLoading,
+  //   error,
+  // } = useExpenses(token, (currentPage - 1) * 10, 10, sortOrder);
+
+  // const toggleSortOrder = () => {
+  //   setSortOrder(sortOrder === "asc" ? "desc" : "asc");
+  // };
+
+  // if (isLoading) return <div>Loading...</div>;
+  // if (error) return <div>An error occurred: {error.message}</div>;
+
+  return (
+    <div className="expenses-list">
+      <ExpenseListHeader
+        listName={list.name}
+        expenseTotal={list.expenseTotal.toFixed(2)}
+      />
+      {/* TODO: */}
+      {/* <Button
+        icon={
+          sortOrder === "asc" ? (
+            <SortAscendingOutlined />
+          ) : (
+            <SortDescendingOutlined />
+          )
+        }
+        onClick={toggleSortOrder}
+      >
+        Sort
+      </Button> */}
+      <ExpensesHeader />
+      <div className="sperated-expenses-list" />
+      {list.expenses?.map((item, index) => (
+        <ExpenseItem
+          key={item._id}
+          item={item}
+          className={index % 2 === 0 ? "expense-item even" : "expense-item odd"}
```

Comment:
css nth child

Reply:
in other branch I will fix it
https://github.com/ofir-zeitoun/expenses-client/issues/33